LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/state-pseudo-class.html

```diff
@@ -106,27 +112,27 @@
   let inner = outer.innerElement;
   let innerStates = inner.i.states;
 
-  innerStates.add('--innerFoo');
+  innerStates.add(':state(innerFoo)');
   assert_equals(getComputedStyle(inner).opacity, '0.5',
-      '::part() followed by :--foo');
-  innerStates.delete('--innerFoo');
-  innerStates.add('--innerfoo');
+      '::part() followed by :state(--foo)');
```

Comment:
Can you change to `'::part() followed by :state(innerFoo)'` or `'::part() followed by :state(foo)'`?
One of the two for consistency